src/main/webapp/src/components/CourseDetails.vue

```diff
@@ -206,20 +253,117 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.label {
-    width: 7rem;
-    font-weight: 700 !important;
-    float: left !important;
+// page container
+.course-details-container {
+    margin: auto;
+    max-width: 1200px;
+}
+.course-details-main-container {
+    display: flex;
+    justify-content: space-between;
+}
+// buttons container
+.course-details-nav {
+    margin-bottom: $xxl-space;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.nav-item > button:first-child {
+    margin-right: $s-space;
+}
+.button-icon {
+    padding-right: $xs-space;
+}
+// course content
+.course-details-content-container {
+    max-width: 640px;
+}
+.course-details-image-container {
+    margin-bottom: $l-space;
+    width: 640px;
+    min-height: 120px;
+    max-height: auto;
+    padding: 3px;
+}
+.course-img {
+    max-width: 100%;
+    height: auto;
+}
+.course-details-content-date {
+    margin-bottom: $xs-space;
+}
+.course-details-content-title {
+    font-size: $xl-font;
+    font-weight: $normal;
+    margin-bottom: $l-space;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-line-clamp: 5;
+    -webkit-box-orient: vertical;
 }
 
-.detailThumbnail {
-    max-width: 20rem;
+.course-details-content-subtitle {
+    font-size: $l-font;
+    font-weight: $normal;
+    margin-bottom: $m-space;
+}
+.course-details-content-text {
+    font-size: $s-font;
+    line-height: 1.57;
+    margin-bottom: $m-space;
+}
+// course summary
+.course-details-summary-container {
+    background: $light-grey;
+    padding: $m-space;
+    text-align: left;
+    width: 320px;
+    align-self: start;
 }
 
-.abbreviation {
-    white-space: nowrap;
+.course-details-summary-title {
+    font-size: $l-font;
+    font-weight: $normal;
+    margin-bottom: $m-space;
+}
+.course-details-summary-label {
+    font-size: $m-font;
+    font-weight: $normal;
+    margin-bottom: $xs-space;
+}
+.course-details-summary-text {
+    font-size: $s-font;
+    font-weight: initial;
+    line-height: 1.57;
+    margin-bottom: $xs-space;
     overflow: hidden;
     text-overflow: ellipsis;
-    width: calc(100% - 7rem);
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+}
+.course-details-summary-icon-text {
+    font-size: $s-font;
+    font-weight: initial;
+}
+.course-details-summary-icon {
+    margin-right: $s-space;
+}
+.course-details-summary-with-icon {
+    display: flex;
+    align-items: center;
+    .comma {
+        margin-right: $xs-space;
+    }
+    .hyphen {
```

Comment:
&.hyphen
?

Reply:
no because I just want to access .comma alone
.comma is a child element of .course-details-summary-with-icon
&.hyphen means` <div class="course-details-summary-with-icon hyphen"/>`
but what we have is 
```
<div class="course-details-summary-with-icon hyphen">
   <div class="hyphen">
          ,
   </div
</div
```